test/v3_test.dart

```diff
@@ -19,6 +19,20 @@ final _endpoint = PgEndpoint(
 final _sessionSettings = PgSessionSettings(
   // To test SSL, we're running postgres with a self-signed certificate.
   onBadSslCertificate: (cert) => true,
+  transformer: StreamChannelTransformer(
+    StreamTransformer.fromHandlers(
+      handleData: (data, sink) {
+        print('[IN]: $data');
```

Comment:
Do we want these print statements in the test?

Reply:
No, my bad 🤦 They're left over from me debugging tests.

Reply:
Similar to what the `docker_process` package is doing, I've replaced them with a logger that is disabled by default. That way, to debug failing tests, it's easy to enable the logs to see where it's going wrong.
If you think that code shouldn't be in tests I can remove it as well.